TRIAGE.md

```diff
@@ -0,0 +1,65 @@
+
+Triage helps ensure issues resolve quickly by:
+
+- Ensuring the issue's intent and purpose is conveyed precisely. This is necessary because it can be difficult for an issue to explain how an end user experiences a problem and what actions they took.
+- Giving a contributor the information they need before they commit to resolving an issue.
+- Lowering the issue count by preventing duplicate issues.
+- Streamlining the development process by preventing duplicate discussions.
+
+If you don't have the knowledge or time to code, consider helping with triage. The community will thank you for saving them time by spending some of yours.
+
+## GitHub Labels
+
+Area labels help categorize issues based on their scope:
+| Label                    | Description                                                               |
+| ------------------------ | ------------------------------------------------------------------------- |
+| `bug`                    | Indicates that the issue is a bug or a software defect.                   |
+| `enhancement`            | Indicates that the issue suggests an improvement or new feature.          |
+| `documentation`          | Indicates improvements or additions to documentation.                     |
+| `roadmap`                | Indicates epics and tasks related to roadmap items.                       |
+| `programs`               | Indicates epics and tasks related to Community programs.                  |
+| `team`                   | Indicates issues for team collaboration.                                  |
+| `adr-required`           | Indicates issues requiring Architectural Decision Record                  |
+
+Severity labels:
+| Label                    | Description                                                               |
+| ------------------------ | ------------------------------------------------------------------------- |
+| `High priority`          | High priority flow or feature is broken/non-functional, crash.            |
+| `Critical priority`      | Indicates Critical impact on data or site: data loss (posts, pages, comments), unexpected publishing, security issue.|
+
+
+
+Other labels:
+| Label                    | Description                                                               |
+| ------------------------ | ------------------------------------------------------------------------- |
+| `good first issue`       | Indicates that the issue is suitable for newcomers to contribute to the project.|
+| `question`               | Indicates that the issue lacks sufficient information to be able to act on it right now and requires additional details from the reporter. The maintainer for the area must remove the tag when more info is provided, or close it if it is too old.|
```

Comment:
I don't think `question` is the right label here. Where we need more infomation, we should use something like `info-required`. I expect Issues which are questions will be moved to Discussions.